backend/src/main/java/de/bund/digitalservice/ris/norms/adapter/output/resource/ResourceService.java

```diff
@@ -0,0 +1,44 @@
+package de.bund.digitalservice.ris.norms.adapter.output.resource;
+
+import de.bund.digitalservice.ris.norms.application.port.output.LoadPortalPublishingAllowListPort;
+import java.io.IOException;
+import java.nio.file.Files;
+import java.util.ArrayList;
+import java.util.List;
+import lombok.extern.log4j.Log4j2;
+import org.springframework.beans.factory.annotation.Value;
+import org.springframework.core.io.Resource;
+import org.springframework.stereotype.Service;
+
+/**
+ * Service for loading data from classpath resources.
+ */
+@Service
+@Log4j2
+public class ResourceService implements LoadPortalPublishingAllowListPort {
+
+  private final List<String> publishingAllowlist = new ArrayList<>();
+
+  public ResourceService(
+    @Value(
+      "classpath:/portal-prototype-publishing-allowlist.txt"
+    ) Resource portalPrototypePublishingAllowlist
+  ) {
+    try {
+      this.publishingAllowlist.addAll(
+          Files.readAllLines(portalPrototypePublishingAllowlist.getFile().toPath())
+        );
+    } catch (IOException e) {
```

Comment:
I think we should raise an exception if we were not able to load the resource which will happen on application start-up. Otherwise we wouldn't realize it and the next prototype job run would not filter by the abbreviations and we would be possibly publishing not-allowed-norms.
What about not catching the `IOException` and just propagating it?

Reply:
The fallback is an empty list and in that case we will not publish anything.
But yes that exception handling is not really needed here, ill remove it (and just propagate the exception) :)

Reply:
Oh you are right, having an empty list would lead to publishing nothing.. but yeah we should know when this happened, otherwise we won't be updating the data on the buckets ever